src/analyze/analyze-security.c

```diff
@@ -1538,6 +1538,7 @@ static int assess(const struct security_info *info, Table *overview_table, Analy
                         return log_error_errno(r, "Failed to output table: %m");
         }
 
+        assert(weight_sum > 0);
```

Comment:
We already have assert on the correct place. This commit is not necessary.

Reply:
Indeed.